app/javascript/dashboard/store/modules/customViews.js

```diff
@@ -86,6 +93,7 @@ export const mutations = {
   [types.SET_CUSTOM_VIEW]: MutationHelpers.set,
   [types.UPDATE_CUSTOM_VIEW]: MutationHelpers.update,
   [types.DELETE_CUSTOM_VIEW]: MutationHelpers.destroy,
+  [types.SET_CUSTOM_VIEW_COUNT]: MutationHelpers.setSingleRecord,
```

Comment:
Should we use `updateAttributes` instead?

Reply:
@scmmishra That fetches the entire record, I have checked in other parts of the code we are using the single record update.

cc @iamsivin

Reply:
Noted